package.json

```diff
@@ -13,6 +13,8 @@
   ],
   "dependencies": {
     "antd": "~4.20.0-alpha.0",
+    "dayjs": "~1.11.1",
+    "rc-picker": "~2.6.8",
```

Comment:
I thought `antd` includes `rc-picker`? 🤔

Reply:
Yes, we should declare explicitly if we import the deps and republish it.